pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -179,9 +211,9 @@ func NewPriorityQueueWithClock(stop <-chan struct{}, clock util.Clock, fwk frame
 	}
 
 	pq := &PriorityQueue{
-		clock:            clock,
+		clock:            options.clock,
 		stop:             stop,
-		podBackoff:       NewPodBackoffMap(1*time.Second, 10*time.Second),
+		podBackoff:       NewPodBackoffMap(1*time.Second, options.podMaxBackoffDuration),
```

Comment:
why don't we make initial wait duration configurable??

Reply:
hmmm, any use case for this ? 
I think 1s makes sense at most cases

Reply:
If we just want to make the backoff policy flexible, maybe the way to calculate backoff duration is also need to be configurable...

Reply:
In the situation which I mentioned in https://github.com/kubernetes/kubernetes/pull/81698,  there are a bunch of unschedulable&high priority pods and a few schedulable&low priority pods,  initial wait duration longer than `1 second`  could prevent from low priority pods starvation effectively.

Reply:
@bsalamat  What do you think of adding a `PodInitialBackoffDuration` to the scheduler config along with the `PodMaxBackoffDuration`?

Reply:
We should bring this to the sig-meeting to get a consensus.

Reply:
Sorry that I couldn't attend to the meeting.  Did the sig meeting get the conclusion??  According to the meeting recording,  nobody seemed to mention to `PodInitialBackoffDuration`.

Reply:
nits: "largar than"  should be "larger than or equal"?? (no backoff)

Reply:
resolved